ipc/src/test/java/com/c8y/ms/templates/ipc/AppTest.java

```diff
@@ -0,0 +1,13 @@
+package com.c8y.ms.templates.ipc;
+
+import org.junit.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class AppTest {
+
+	@Test
+	public void contextLoads() {
+	}
```

Comment:
maybe it would be good to have a least a simple test here? otherwise this class is quite useless...

Reply:
it checks at least if the spring context gets loaded, if something is wrong test will fail. This is just a template not productive code, in my point of view ok without tests.